src/test/regress/expected/multi_complex_expressions.out

```diff
@@ -347,3 +347,84 @@ SELECT count(*) FROM lineitem JOIN orders ON l_orderkey = o_orderkey
 SELECT count(*) FROM lineitem, orders WHERE l_orderkey + 1 = o_orderkey;
 ERROR:  cannot perform local joins that involve expressions
 DETAIL:  local joins can be performed between columns only
+-- Check that we can issue limit/offset queries
+-- SIMPLE OFFSET without LIMIT and ORDER BY
+SELECT o_orderkey FROM orders WHERE o_orderkey > 9000 AND o_orderkey < 9100 OFFSET 10;
```

Comment:
Are we sure that this will give the same result all the time without an order by?